static/styles/zulip.css

```diff
@@ -1055,12 +1064,16 @@ td.pointer {
     }
 
     .edit_content {
-        width: 12px;
+        width: 20px;
```

Comment:
Why does this need to change?

Reply:
With the new icon, the gap between the `pencil/source code` and `quote and reply` looks smaller as compared to the gap between the others. To ensure the UI remains consistent, I changed it to 20px.

**with 12px:**
![Screenshot from 2023-02-03 09-04-53](https://user-images.githubusercontent.com/66828942/216507009-65bff6b4-ffb8-4a68-b604-e7aba73e9ee2.png)
![Screenshot from 2023-02-03 09-05-01](https://user-images.githubusercontent.com/66828942/216507018-01daf178-6347-4802-ba21-ddeba0225b48.png)

**with 20px:**
![Screenshot from 2023-02-03 09-05-12](https://user-images.githubusercontent.com/66828942/216507048-f21a7094-50fa-4fb5-ad8c-8f5e94094cc7.png)
![Screenshot from 2023-02-03 09-05-22](https://user-images.githubusercontent.com/66828942/216507063-49f2bc1f-6403-4fa3-b933-4c1e15de9630.png)